apps/percona/chaos/openebs_volume_replica_failure/test.yml

```diff
@@ -10,7 +10,7 @@
 
         ## DERIVE THE APP STORAGE CLASS AND CHAOS UTIL TO USE       
  
-        - include: test_prerequisites.yaml
+        - include: test_prerequisites.yml
   
         - include_vars:
             file: chaosutil.yaml
```

Comment:
Extension needs to be changes. In prerequisites, it is .yml

Reply:
@shashank855 , the file is still named .yaml -> https://github.com/openebs/litmus/blob/master/apps/percona/chaos/openebs_volume_replica_failure/test_prerequisites.yaml

Reply:
@ksatchit  I have made that change.